libs/Model/Model.cpp

```diff
@@ -104,38 +93,34 @@ void Model::setBoundingVolumenRadius(float val)
 	m_bounding_volumen_radius = val;
 }
 
-bool Model::RayInstersection(const glm::vec3& start, const glm::vec3& rayDirection)
+bool Model::RayInstersection(const glm::vec3 &start, const glm::vec3 &rayDirection)
 {
-  
-  glm::vec3 Position = m_position;
 
-  float b = 2.f * (rayDirection.x * (start.x - Position.x) +
-    rayDirection.y * (start.y - Position.y) + rayDirection.z * (start.z - Position.z));
-  float c = start.x * start.x - 2.f * start.x * Position.x + Position.x * Position.x
-    + start.y * start.y - 2.f * start.y * Position.y + Position.y * Position.y
-    + start.z * start.z - 2.f * start.z * Position.z + Position.z * Position.z - m_bounding_volumen_radius * m_bounding_volumen_radius;
+	glm::vec3 Position = m_position;
+
+	float b = 2.f * (rayDirection.x * (start.x - Position.x) +
+					 rayDirection.y * (start.y - Position.y) + rayDirection.z * (start.z - Position.z));
+	float c = start.x * start.x - 2.f * start.x * Position.x + Position.x * Position.x + start.y * start.y - 2.f * start.y * Position.y + Position.y * Position.y + start.z * start.z - 2.f * start.z * Position.z + Position.z * Position.z - m_bounding_volumen_radius * m_bounding_volumen_radius;
 
-  float discr = (b * b - 4.f * c);
-  if (discr < 0.f)
-  {
-    m_selected = false;
-    
-  }
-  else
-  {
-    m_selected = true;
-    
-  }
+	float discr = (b * b - 4.f * c);
+	if (discr < 0.f)
+	{
+		m_selected = false;
+	}
```

Comment:
It seems like sometimes the tabbing is 4 spaces and other times it's 2. Is there a reason for this?

Reply:
I reformatted the file